dotnet/Cucumber.Messages.Specs/BasicMessageSerializationTests.cs

```diff
@@ -0,0 +1,36 @@
+using Xunit;
+﻿using System.IO;
+using Io.Cucumber.Messages.Types;
+using System.Text.Json;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text.Json.Serialization;
+using System;
+using System.ComponentModel;
+using Cucumber.Messages;
+
+namespace Cucumber.Messages.Specs
+{
+    public class BasicMessageSerializationTests
+    {
+
+        [Fact]
+        public void SerializesAnEnvelopeToNDJSONCorrectly()
+        {
+            var stepDefinitionNDJSON = @"{""stepDefinition"":{""id"":""0"",""pattern"":{""source"":""I have {int} cukes in my belly"",""type"":""CUCUMBER_EXPRESSION""},""sourceReference"":{""location"":{""line"":3},""uri"":""samples/minimal/minimal.feature.ts""}}}
+";          
+            var soureReference = new SourceReference("samples/minimal/minimal.feature.ts",
```

Comment:
sourceReference*

Reply:
Fixed misspelled variable name.

Re: the comment above about "Is this still needed?" - does that refer to the dependabot itself or the use of the devcontainer?

Reply:
The use of dependabot. As I know we were discussing using renovate which is used most other places.